src/pages/Book/book.styles.js

```diff
@@ -2,9 +2,22 @@ import styled from "styled-components";
 
 export const Container = styled.div`
   display: inline;
-  margin: 10px;
+  margin-left: 20px;
 `;
 
 export const CoverDiv = styled.div`
   display: inline;
 `;
```

Comment:
if you want an inline element you can use a span, this is not wrong, but a span has a default to `display: inline`